.bumpversion.cfg

```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = False
 tag = False
-current_version = 0.1.0
+current_version = 0.1.1
```

Comment:
Why are release changes here? They are already in the master branch

Reply:
May be as a result of rebase (git pull from master)?

Reply:
Changed: from `current_version = 0.1.1` to `current_version = 0.1.0` in .bumpversion.cfg

Reply:
Please don't change release versions. I think rebase on master with picking only necessary commits should be enough